packages/backend/API.md

```diff
@@ -70,7 +71,7 @@ export { ConstructFactoryGetInstanceProps }
 export { defineAuth }
 
 // @public
-export const defineBackend: <T extends DefineBackendProps>(constructFactories: T) => Backend<T>;
+export const defineBackend: <T extends DefineBackendProps>(constructFactories: T, mainStackProps?: MainStackProps) => Backend<T>;

```

Comment:
We should prepare this API for further expansion. I.e. main stack props might not be the only extension we do in the future.

Therefore, we should have `defineBackend(constructFactories T, props?: DefineBackendProps)`
and then `DefineBackendProps` should have `mainStackProps: MainStackProps` as optional property.

Reply:
Have updated in #[93b77dc](https://github.com/aws-amplify/amplify-backend/pull/1675/commits/93b77dcadb1e2d8531e8492fd91ccf2fa86f621e).